velero-plugins/util/openshift/build.go

```diff
@@ -0,0 +1,41 @@
+package openshift
+
+import (
+	"context"
+
+	"github.com/konveyor/openshift-velero-plugin/velero-plugins/clients"
+	buildv1 "github.com/openshift/api/build/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func IsBuildPod(name, namespace string) (bool, error) {
+	cc, err := clients.CoreClient()
+	if err != nil {
+		return false, err
+	}
+	pod, err := cc.Pods(namespace).Get(context.Background(), name, metav1.GetOptions{})
+	if err != nil {
+		return false, err
+	}
+	return isBuildPod(pod), nil
+}
+
+// Check if pod is a build pod via it's annotations
+// If used in RIA, use unmarshal from input.ItemFromBackup because input.Item may have metadata reset
+func PodIsBuildPod(pod *corev1.Pod) bool {
+	return isBuildPod(pod)
+}
+
+// https://github.com/openshift/openshift-controller-manager/blob/aabcbc2cf5d944f64e6ebdbc4e0ce7f1b95bd127/pkg/build/controller/build/build_controller.go#L2549
+func isBuildPod(pod *corev1.Pod) bool {
```

Comment:
would change functions names, only difference is the case

Reply:
I think that's a fairly common pattern if there's an exported func and an internal one. YOu could call it `isBuildPodInternal`, but I don't think that's necessary.

Reply:
I'll keep this one the same as the intention is to copy over private funcs from openshift controller. You want the other one changed?

Reply:
resolved in f6f6c15d4ce9ed80f6d8aa00521a0641f16c5ca4